sources/packages/api/src-sql/Restrictions/Add-col-action-type-and-notification-type.sql

```diff
@@ -0,0 +1,15 @@
+-- Add action_type to restrictions table.
+ALTER TABLE
+  sims.restrictions
+ADD
+  COLUMN IF NOT EXISTS action_type sims.restriction_action_type [] NOT NULL DEFAULT ARRAY ['No effect'] :: sims.restriction_action_type [];
```

Comment:
Is default action type required? As we are truncating we can remove this default.